setup.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020-2024 Quantinuum
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+
+from setuptools import setup  # type: ignore
+
+setup(
+    name="quantinuumqircheck",
+    version="0.1.0",
+    author="TKET development team",
+    author_email="tket-support@quantinuum.com",
+    python_requires=">=3.10",
+    project_urls={
+        "Documentation": "https://tket.quantinuum.com/extensions/pytket-qir/index.html",
+        "Source": "https://github.com/CQCL/quantinuum-qircheck",
+        "Tracker": "https://github.com/CQCL/quantinuum-qircheck/issues",
+    },
+    description="Package to check QIR for compatibility with quantinuum devices",
+    long_description=(Path(__file__).parent / "README.md").read_text(),
```

Comment:
Not sure we want the README, which includes detailed instructions for developers, to appear on the pypi page. Could make a separate file for the long description, or omit.

Reply:
done in [remove setup.py, add project.toml](https://github.com/CQCL/quantinuum-qircheck/pull/1/commits/86416e6b9689510edbba56823c6597b843c5f653)
[86416e6](https://github.com/CQCL/quantinuum-qircheck/pull/1/commits/86416e6b9689510edbba56823c6597b843c5f653)